native/CMakeLists.txt

```diff
@@ -1518,6 +1639,19 @@ cocos_source_files(
     NO_WERROR    cocos/bindings/auto/jsb_scene_auto.cpp
                  cocos/bindings/auto/jsb_scene_auto.h
 )
+endif()
+if(OPENHARMONY)
+    cocos_source_files(
+        NO_WERROR    cocos/bindings/auto/jsb_gfx_auto.cpp
+                     cocos/bindings/auto/jsb_gfx_auto.h
+        NO_WERROR    cocos/bindings/auto/jsb_pipeline_auto.cpp
+                     cocos/bindings/auto/jsb_pipeline_auto.h
+        NO_WERROR    cocos/bindings/auto/jsb_scene_auto.cpp
+                     cocos/bindings/auto/jsb_scene_auto.h
+        NO_WERROR    cocos/bindings/auto/jsb_cocos_auto.cpp
+                     cocos/bindings/auto/jsb_cocos_auto.h
```

Comment:
Put the same auto-generated files into a group instead of copying most of them.

```
cocos_source_files(
        NO_WERROR    cocos/bindings/auto/jsb_gfx_auto.cpp
                     cocos/bindings/auto/jsb_gfx_auto.h
        NO_WERROR    cocos/bindings/auto/jsb_pipeline_auto.cpp
                     cocos/bindings/auto/jsb_pipeline_auto.h
        NO_WERROR    cocos/bindings/auto/jsb_scene_auto.cpp
                     cocos/bindings/auto/jsb_scene_auto.h
       ...
       ...
```

if( NOT OPENHARMONY)
 more...
endif()

Reply:
At present, it has not been adapted yet (it is not completely unsupported). If it is supported later, it will be reopened. It doesn't matter at present, if it is supported in the future, we can delete this piece completely.